packages/fast-usdc/test/cli/airdrop-commands.test.js

```diff
@@ -0,0 +1,245 @@
+    proof,
+    tier,
+    pubkey,
+  },
+});
+
+const trace = label => value => {
+  console.log(label, '::::', value);
+  return value;
+};
+const traceAsync = label => value => Promise.resolve(value).then(trace(label));
+
+const composeM =
+  method =>
+  (...ms) =>
+    ms.reduce((f, g) => x => g(x)[method](f));
+
+const composePromises = composeM('then');
+
+// Successfully
+const collectPowers = config =>
+  composePromises(
+    trace('inspecting walletKit capabilities'),
+    handleMakeSmartWalletKit(config),
+    traceAsync('network config'),
+    () => handleFetchNetworkConfig({ env: config.env, fetch: config.fetch }),
+  );
+
+const addAirdropCommands = (program, { fetch, stderr, stdout }) => ({
+  operator: program.command('claimer').description('Claim airdrop commands'),
+  invoke() {
+    this.operator
+      .command('send')
+      .description(
+        'constructs an OfferSpec in accordance with makeClaimTokensInvitation from ertp-airdrop.',
+      )
+      .option('--proof <string>', 'Proof', String)
+      .option('--pubkey <string>', 'Public key', String)
+      .option('--tier <number>', 'Tier', Number)
+      .action(async opts => {
+        const { proof, tier, pubkey } = opts;
+        const powers = await collectPowers({
+          delay: 500,
+          fetch,
+          env: { AGORIC_NET: 'xnet' },
+        })();
+
+        const [instance, brand] = await getInstanceAndBrand({
+          brandName: 'IST',
+          instanceName: 'tribblesXnetDeployment',
+        })(powers);
+        console.log('------------------------');
+        console.log({ instance, brand });
+
+        /** @type {import('@agoric/smart-wallet/src/invitations.js').InvitationSpec} */
+        const offer = makeClaimAirdropOffer(
+          { feeBrand: brand, instanceName: 'tribblesXnetDeployment' },
+          { proof, tier, pubkey },
+        );
+
+        console.log({ offer });
+        outputActionAndHint(
+          { method: 'executeOffer', offer },
+          { stderr, stdout },
+          powers.marshaller,
+        );
+      });
+    return this.operator;
+  },
+});
+
+export const delay = ms => new Promise(resolve => setTimeout(resolve, ms));
+
+// Alternative: Async pipe executes functions left to right (sometimes more intuitive)
+
+test('tribbles airdrop program', async t => {
+  const program = new Command('Airdrop');
+  const claimArgs = harden(offerArgs);
+  const { proof, pubkey, tier } = claimArgs;
+
+  const stringProof = JSON.stringify(proof);
+
+  const argv = [
+    ...`node Airdrop claimer send`.split(' '),
+    ...flags({ proof: stringProof, pubkey, tier }),
+  ];
+  t.log(...argv);
+  program.exitOverride();
+  const out = [];
+  const err = [];
+
+  addAirdropCommands(program, {
+    fetch: fetch,
+    stdout: mockStream(out),
+    stderr: mockStream(err),
+    env: {},
+    now: () => 1234,
+  }).invoke();
+  await program.parseAsync(argv);
+
+  const action = marshalData.fromCapData(JSON.parse(out.join('')));
```

Comment:
reason for this test assertion is that it existed in the *commands test file I originally copied as my starting point. The result of this logic is the following error:

```sh
 Error {
    message: 'internal: decodeRemotableFromSmallcaps option must return a remotable: "board0257"',
  }
 ```